hack/generator/pkg/jsonast/jsonast.go

```diff
@@ -552,7 +543,7 @@ func allOfHandler(ctx context.Context, scanner *SchemaScanner, schema *gojsonsch
 		case *astmodel.ResourceType:
 			// it is a little strange to merge one resource into another with allOf,
 			// but it is done and therefore we have to support it.
-			// (an example is Microsoft.VisualStudio’s Project type)
+			// (an example is Microsoft.VisualStudio's Project type)
```

Comment:
boo 🙂

Reply:
I think this actually arose from my copying this file into vscode and back (possibly across my VM boundary?) which I had to do because of some shenanigans unrelated to this change.

I'll put this back =)

Reply:
Actually I just reformatted it to what Bevan suggested below. I am not philisophically opposed to the original character though it just got mangled in some horrible copy operation I did and I replaced it manually thinking that's what it had been when it reality it was ’ not '